src/pretix/settings.py

```diff
@@ -116,7 +116,7 @@
 
 MEDIA_URL = config.get('urls', 'media', fallback='/media/')
 
-PRETIX_INSTANCE_NAME = config.get('pretix', 'instance_name', fallback='pretix.de')
+PRETIX_INSTANCE_NAME = config.get('pretix', 'instance_name', fallback='eventyay')
```

Comment:
Can this be changed from "PRETIX_INSTANCE_NAME" to "NSTANCE_NAME" and the following items as well?

If this is a big complicated please open a separate issue about this.